examples/cfd/vortex_shedding_mgn/inference.py

```diff
@@ -156,6 +156,12 @@ def predict(self):
             self.faces.append(torch.squeeze(cells).numpy())
             self.graphs.append(graph.cpu())
 
+    def get_raw_data(self, idx):
+        self.pred_i = [var[:, idx] for var in self.pred]
+        self.exact_i = [var[:, idx] for var in self.exact]
+
+        return(self.graphs, self.faces, self.pred_i, self.exact_i)
```

Comment:
Add space and parentheses are not needed.

Reply:
Fixed!